Fishing Game/ViewControllers/FishingReel.swift

```diff
@@ -116,7 +116,7 @@ class FishingReel {
     // Uses one bait when a fish bites
     func useBait() {
         print("Used bait")
-        
+        guard fishingDay?.tacklebox.baitCount != 0 else { return }
```

Comment:
This is probably fine, but TECHNICALLY this should be a check for > 0 since this would pass if the bait count was negative.